src/main/java/com/lob/model/Check.java

```diff
@@ -41,7 +42,7 @@
     @JsonProperty private final List<TrackingEvent> trackingEvents;
     @JsonProperty private final List<Thumbnail> thumbnails;
     @JsonProperty private final String mailType;
-    @JsonProperty private final Map<String, String> mergeVariables;
+    @JsonProperty private final Map<String, Object> mergeVariables;

```

Comment:
Changing this type spec to `Map<String, Object>` because merge variables values can be nested objects, lists, booleans. I think the `Object` type encompasses all of those types. Well at least the compiler doesn't complain about it.